index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!-- NAVBAR HERE -->
+    <nav>
```

Comment:
- I think it would be better to use a semantic tag header on the part of your page that contains the logo and nav bar
- please wrap this section in a heading tag 
- By doing this you will be following the rules of HTML 5 tags. the header for the header section, nav for navigation bar etc
```suggestion
    <header> 
    </header>
```
![image](https://user-images.githubusercontent.com/59427923/198932706-8c383fce-6b0e-40c7-b0e4-204a6bd82d73.png)